src/script/lua_api/l_base.cpp

```diff
@@ -84,3 +85,15 @@ bool ModApiBase::registerFunction(lua_State *L, const char *name,
 
 	return true;
 }
+
+/**
+ * A number is Nan if rawequals of itself return false
```

Comment:
Outdated description. The entire doxygen comment looks superfluous how it currently is.